src/version.cpp.in

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2017, The Monero Project
```

Comment:
Please remove all the comment headers from this file. As a generated file, they're pointless. And since we're regenerating it every time, smaller is better.

Reply:
ok